magma/port.py

```diff
@@ -208,7 +208,7 @@ def trace(self):
             if len(self.wires.inputs) < 1:
                 # print('Warning:', str(self), 'is not connected to an input')
                 return None
-            assert len(self.wires.inputs) == 1
+            #assert len(self.wires.inputs) == 1
```

Comment:
Is this necessary for this change set? This logic is a bit weird, and will be cleared up in the `hwtypes` branch.  I guess technically this shouldn't break any existing code (since it would've triggered an assertion failure otherwise), but there may be other internal code that depends on this assumption (this is just my assumption though, I'm not sure if that's true)

Reply:
That's right, uncommented it.

(This might be a spurious assertion though, calling `wired()` on an input that drives > 1 wire doesn't work due to this assertion...can take a look later.)